### PR TITLE
Deprecate archer v1

### PR DESCRIPTION
**Pull request by adi88d**
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

File: Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml [hunk 1]
```diff
@@ -2,11 +2,11 @@ commonfields:
   id: RSA Archer
   version: -1
 name: RSA Archer
-display: RSA Archer
+display: RSA Archer - Deprecated
```
Comment (guykeller) [inline Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml:5]: @ChanochShayner does this match the standard we're enforcing with your new dev task?

Comment (ChanochShayner) [reply Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml:5]: No, The display_name (display) of all deprecated integrations should end with (Deprecated).

Comment (guykeller) [reply Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml:5]: @adi88d @yaakovi FYI

Comment (ChanochShayner) [reply Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml:5]: Also, the description of all deprecated integrations should start with "Deprecated."

File: Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml [hunk 2]
```diff
@@ -2,11 +2,11 @@ commonfields:
...
-description: The RSA Archer GRC Platform provides a common foundation for managing
-  policies, controls, risks, assessments and deficiencies across lines of business.
+description: Deprecated - We recommend using RSA Archer v2 instead.
```
Comment (yaakovi) [inline Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml:9]:
```suggestion
description: Deprecated. We recommend using RSA Archer v2 instead.
```

File: Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml (hunk 1 above)
Comment (yaakovi) [inline Packs/ArcherRSA/Integrations/integration-ArcherRSA.yml:5]:
```suggestion
display: RSA Archer (Deprecated)
```

File: Packs/ArcherRSA/ReleaseNotes/1_1_1.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### RSA Archer - Deprecated
+- We recommend using RSA Archer v2 instead.
```
Comment (yaakovi) [inline Packs/ArcherRSA/ReleaseNotes/1_1_1.md:4]:
```suggestion
##### RSA Archer
Deprecated. We recommend using RSA Archer v2 instead.

```

**Review by yaakovi — CHANGES_REQUESTED** [review]
add `deprecated: true` to all integration commands as well.

**Comment by guykeller** [comment]
@adi88d please merge from master before merging this to master, to make sure it passes the new validation for deprecated packs

File: Packs/ArcherRSA/ReleaseNotes/1_1_1.md (hunk 3 above)
Comment (adi88d) [reply Packs/ArcherRSA/ReleaseNotes/1_1_1.md]: i changed it to:

```

#### Integrations
##### RSA Archer (Deprecated)
Deprecated. We recommend using RSA Archer v2 instead.

```
after gets this error in validate:
`Packs/ArcherRSA/ReleaseNotes/1_1_1.md: [RN107] - No release note entry was found for the integration "RSA Archer (Deprecated)" in the ArcherRSA pack.`

**Review by yaakovi — APPROVED** [review]

**Comment by Itay4** [comment]
@adi88d what is the status here?

**Comment by adi88d** [comment]
> @adi88d what is the status here?

The build failed in `Archer v2 - Test`, there is some problem with the instance, im on it

**Pull request merged**